src/Sulu/Bundle/AdminBundle/Resources/translations/admin.de.json

```
@@ -31,6 +31,7 @@
     "sulu_admin.no": "Nein",
     "sulu_admin.type": "Typ",
     "sulu_admin.default": "Standard",
+    "sulu_admin.locale": "Locale",
```

maybe we should go with:

```suggestion
    "sulu_admin.language": "Language",
```